project_management/workshop_advert.md

```diff
@@ -0,0 +1,56 @@
+## Boost your research reproducibility with Binder
+
+If you are looking for an easy way to share your code with others, without worrying about what software they have, then myBinder.org might be the solution for you. Binder allows you to share your software with others in the form of a single clickable link, making it easier for users and removing the need for you to worry about supporting lots of different platforms. 
+
+During this free workshop we will discuss reproducible computing environments, show examples of others’ projects in myBinder.org and help you learn how to prepare a binder-ready project. At the end of the workshop you will be able to take some of your own content (in a R or Jupyter notebook) and prepare it so that it can be used by others on myBinder.org.
+
+This workshop is for people who are:
+*	Interested in reproducibility, containers, Docker or continuous integration; 
+*	Already familiar with R Markdown or Jupyter notebooks;
```

Comment:
Again I wouldn't say this is necessarily a requirement, we can work with scripts as well.

Reply:
(I'm working up from the bottom, see my comment below).

I think for this workshop we will demo an R Markdown file and a Jupyter notebook in the morning. If folks don't know either then they'll be quite far behind and we won't have time to catch them up.

I think "familiar" is pretty light too, so if you work with python scripts you probably can read a jupyter notebook....and that's the requirement.